optlib/inko.ctags

```diff
@@ -0,0 +1,57 @@
+--langdef=Inko
+--map-Inko=+.inko
+--kinddef-Inko=o,object,Object definition
+--kinddef-Inko=m,method,Method definition
+--kinddef-Inko=t,trait,Trait definition
+--kinddef-Inko=a,attribute,Attribute definition
+--kinddef-Inko=c,constant,Constant definition
+--kinddef-Inko=r,reopen,Reopen object
+
+--_fielddef-Inko=implements,Trait being implemented
```

Comment:
`ctags` has a built in field named `inherits`.
`inherits` is not suitbale here?

Java has an interface. However, it uses inherits to store the "implements" information to "inherits" lile:


```
[yamato@slave tmp]$ cat /tmp/foo.java
cat /tmp/foo.java

class foo extends Object implements Cloneable {
}
[yamato@slave tmp]$ u-ctags -o - --fields='*' foo.java 
u-ctags -o - --fields='*' foo.java 
foo	foo.java	/^class foo extends Object implements Cloneable {$/;"	kind:class	line:2	language:Java	inherits:Object,Cloneable	roles:def	end:3
```

Reply:
I suppose this depends a bit on personal tastes. When a trait is implemented, it is not inherited from. Instead, any default methods it may provide are copied into the object that implements the trait. This means "implements" would be more accurate than "inherits".

Reply:
I see. What you wrote may be applicable to other languages. I wonder what I should do.
Anyway, it is nothing to do with your parser.